### DISCUSSION TITLE
fVDB build failing

### DISCUSSION THREAD

**Discussion by DanielHabib** [AcademySoftwareFoundation/openvdb#1948] [Q&A]
I've been trying to build fVDB for 2 weeks now and I keep running into issues. (latest build issues can be seen [here](https://gist.githubusercontent.com/DanielHabib/b3c9818ea8bc57324cf61a1a8be2b832/raw/ff5b73d54c7f560a0367d83bb8a2fe1443c84557/gistfile1.txt) , not sure how to debug without a clear error)
Following the docker instructions doesn't work, and following the conda install instructions also has issues. 

Has anyone been able to reliably build fVDB on Ubuntu recently?

**Comment by swahtz** [comment]
Hi @DanielHabib, sorry I haven't seen these discussions/issues posted until recently.  Yes, so I believe this error has to do with fVDB being built for a compute architecture <8.0 and we're trying to support 8.0 and above (which is the platform supported by Ampere and above).  Are you able to use that compute capability?
